cdap-ui/app/features/tracker/templates/integrations.html

```diff
@@ -0,0 +1,154 @@
+  the License.
+-->
+
+<div class="integrations-container">
+  <div class="row integration-item">
+
+    <div class="integration-information">
+      <h3>Cloudera Navigator</h3>
+      <p>some definition of Cloudera Navigator</p>
+    </div>
+
+    <div class="integration-setup text-right" ng-if="!IntegrationsController.navigatorSetup.isSetup">
+      <button class="btn btn-default btn-setup"
+              ng-click="IntegrationsController.navigatorSetup.isOpen = true">
+        Setup
+      </button>
+
+      <div class="integration-setup-popup text-left"
+           ng-if="IntegrationsController.navigatorSetup.isOpen">
+        <h3>Configuration</h3>
+        <h4>
+          Cloudera
+        </h4>
+        <form class="form-horizontal">
+          <div class="form-group">
+            <label class="control-label col-xs-3">Hostname</label>
+            <div class="col-xs-9">
+              <input type="text" class="form-control"
+                     ng-model="IntegrationsController.navigatorInfo.hostname">
+            </div>
+          </div>
+
+          <div class="form-group">
+            <label class="control-label col-xs-3">Username</label>
+            <div class="col-xs-9">
+              <input type="text" class="form-control"
+                     ng-model="IntegrationsController.navigatorInfo.username">
+            </div>
+          </div>
+
+          <div class="form-group">
+            <label class="control-label col-xs-3">Password</label>
+            <div class="col-xs-9">
+              <input type="password" class="form-control"
+                     ng-model="IntegrationsController.navigatorInfo.password">
+            </div>
+          </div>
+        </form>
+
+        <h4>
+          Kafka
+        </h4>
+        <form class="form-horizontal">
+          <div class="form-group">
+            <label class="control-label col-xs-3">Broker List</label>
+            <div class="col-xs-9">
+              <input type="text" class="form-control"
+                     ng-model="IntegrationsController.navigatorInfo.brokerString">
+            </div>
+          </div>
+        </form>
+
+        <div>
+          <button class="btn btn-default pull-left"
+                  ng-click="IntegrationsController.navigatorSetup.isOpen = false">
+            Cancel
+          </button>
+          <button class="btn btn-success pull-right"
+                  ng-click="IntegrationsController.saveNavigatorSetup()">
+            Save
+          </button>
+        </div>
+      </div>
+    </div>
+
+    <div class="integration-status" ng-if="IntegrationsController.navigatorSetup.isSetup">
+      <div class="status status-item">
+        <p>
+          <span>Status: </span>
+          <span class="fa fa-circle"
+                ng-class="{ 'text-success': IntegrationsController.navigatorState.status === 'RUNNING', 'text-danger': IntegrationsController.navigatorState.status !== 'RUNNING' }"></span>
```

Comment:
Would we possibly have 'COMPLETED' as status? If yes we might want to avoid 'text-danger' for it. But if apart from 'RUNNING' if all other status needs to be red then thats fine.